src/Kentico.Xperience.Lucene.Core/Indexing/DefaultLuceneTaskLogger.cs

```diff
@@ -176,13 +182,16 @@ private async Task HandleEventInternal(IndexEventWebPageItemModel webpageItem, s
 
     private static LuceneTaskType GetTaskType(string eventName)
     {
-        if (eventName.Equals(WebPageEvents.Publish.Name, StringComparison.OrdinalIgnoreCase))
+        if (eventName.Equals(WebPageEvents.Publish.Name, StringComparison.OrdinalIgnoreCase)
+            || eventName.Equals(ContentItemEvents.Publish.Name, StringComparison.OrdinalIgnoreCase))
         {
             return LuceneTaskType.UPDATE;
         }
 
         if (eventName.Equals(WebPageEvents.Delete.Name, StringComparison.OrdinalIgnoreCase) ||
-            eventName.Equals(WebPageEvents.Unpublish.Name, StringComparison.OrdinalIgnoreCase))
+            eventName.Equals(WebPageEvents.Unpublish.Name, StringComparison.OrdinalIgnoreCase) ||
+            eventName.Equals(ContentItemEvents.Unpublish.Name, StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Should be this event name checked twice (ContentItemEvents.Unpublish.Name)? What about ContentItemEvents.Delete and Archive events?